extra/linux/71-liquidctl.rules

```diff
@@ -17,6 +17,12 @@
 # group is not used by default to avoid generating warnings on systems that
 # have already removed it.
 
+# NVIDIA ASUS ROG Strix 2080ti Rev A. GPU
+KERNEL=="i2c-[0-9]*", ATTRS{vendor}=="10de", ATTRS{subsystem_vendor}=="1043", ATTRS{device}=="1e07", ATTRS{subsystem_device}=="866a", TAG+="uaccess"
+
+# NVIDIA EVGA GTX 1080 FTW
+KERNEL=="i2c-[0-9]*", ATTRS{vendor}=="10de", ATTRS{subsystem_vendor}=="3842", ATTRS{device}=="1b80", ATTRS{subsystem_device}=="6286", TAG+="uaccess"
+
```

Comment:
On my system I need to include a `0x` prefix when matching the vendor/device IDs.

```
ATTRS{vendor}=="0x10de"
                ^^
```

Are you running these exact rules?  If so, then it seems there was a change in the kernel since yours and mine (5.9.9).

For reference, here's the relevant part of the output from `udevadm info --name=<...> --attribute-walk`:

```
  looking at parent device '/devices/pci0000:00/0000:00:01.0/0000:01:00.0':
    ...
    ATTRS{device}=="0x1b80"
    ATTRS{subsystem_device}=="0x6286"
    ATTRS{subsystem_vendor}=="0x3842"
    ATTRS{vendor}=="0x10de"
```

Reply:
It definitely worked for me this those exact rules, I just tested and it did not, but it does work for me with the prefix so I can update that to include it.